dkg/dkg.go

```diff
@@ -48,6 +48,55 @@ type Config struct {
 	TestSigning bool
 }
 
+// exchanger is responsible for exchanging partial signatures between peers on libp2p.
+type exchanger struct {
+	setChan chan core.ParSignedDataSet
+	sigex   *parsigex.ParSigEx
+}
+
+func newExchanger(tcpNode host.Host, peerIdx int, peers []peer.ID) exchanger {
+	ex := exchanger{
+		sigex:   parsigex.NewParSigEx(tcpNode, peerIdx, peers),
+		setChan: make(chan core.ParSignedDataSet, len(peers)),
```

Comment:
not sure about `len(peers)`, since:
  - peers can send duplicate messages
  - peers can send different types of messages (lock hash AND deposit datas)

Reply:
so what should be the ideal length of the buffer?